src/sidebar/components/test/login-control-test.js

```diff
@@ -102,23 +101,15 @@ describe('loginControl', function () {
      * disabled, and null if no user profile button is rendered.
      */
     function isUserProfileButtonEnabled(page) {
-      var enabledUserProfileButton  = page.userProfileButton;
-      var disabledUserProfileButton = page.disabledUserProfileButton;
-
-      assert.isTrue(
-        enabledUserProfileButton === null || disabledUserProfileButton === null,
-        'It should never show both the enabled and disabled buttons at once'
-      );
-
-      if (enabledUserProfileButton) {
-        return true;
+      if (page.userProfileButton === null) {
+        return null;
       }
 
-      if (disabledUserProfileButton) {
-        return false;
+      if (page.userProfileButton.classList.contains('is-enabled')) {
```

Comment:
Since this function returns a boolean, you could just return the result of `...contains('...')` directly.